ivy/__init__.py

```diff
@@ -351,27 +351,6 @@ def __sizeof__(self):
     def __dir__(self):
         return self._shape.__dir__()
 
-    def __pow__(self, power, modulo=None):
```

Comment:
Not sure about this, were these methods present in native shape classes like `torch.Size` and `tf.TensorShape`? I'll request @Aarsh2001's review if these functions need to be implemented. Thanks @soma2000-lang 🙂

Reply:
@vedpatwardhan No these were not there as seen in 
![image](https://github.com/unifyai/ivy/assets/56045049/179c71ba-42be-4444-9507-8db01faff207)
thats why I removed it.

Reply:
![image](https://github.com/unifyai/ivy/assets/56045049/3adf2bad-54b4-42bd-97e4-93aa3c3af041)